test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java

```diff
@@ -45,8 +45,8 @@
 	public static OptionalInt threadNumber = OptionalInt.empty();
 	public static OptionalInt hcsSubmitMessage = OptionalInt.empty();
 	public static OptionalInt hcsSubmitMessageSizeVar = OptionalInt.empty();
-	/** initial balance of payer account used for paying for performance test transactions */
-	public static OptionalLong initialBalance = OptionalLong.of(90_000_000_000_000L);
+	/** initial balance of account used as sender for performance test transactions */
+	protected static OptionalLong initialBalance = OptionalLong.of(ONE_HBAR * 9000);
```

Comment:
We wanted 1 mil hbars (`ONE_HBAR * 1_000_000L`) to avoid recharging during load tests. And we still want a public scope to be used in `CryptoCreateForSuiteRunner`